catch (const AnyError& e) {
    std::cout << e << "\n";
}